tests/fsharpqa/Source/test.lst

```diff
@@ -40,8 +40,9 @@ CompilerOptions01,NoMT				CompilerOptions\fsc\debug
 CompilerOptions01,NoMT				CompilerOptions\fsc\dumpAllCommandLineOptions
 CompilerOptions01,NoMT				CompilerOptions\fsc\flaterrors
 CompilerOptions02,NoMT				CompilerOptions\fsc\gccerrors
-CompilerOptions01,NoMT,help				CompilerOptions\fsc\help
+ompilerOptions01,NoMT,help				CompilerOptions\fsc\help
```

Comment:
This should probably be reverted, I assume it is for local testing.

Reply:
@smoothdeveloper, good eyes.  I inadvertently deleted that.

And yes, I had also considered adding a generic error message, it makes a lot of sense.

Reply:
> Could we consider also supporting a method returning a generic error that would make sure our messages for "the version 'current-version' doesn't support feature 'New Feature', this feature is supported in following versions: ..." remain consistent by default and easy to produce in the compiler.

@smoothdeveloper ,
Okay I have spent some time thinking about this.  And I now realize that in fact, it is not an error for the feature to be not supported by the specified version of the compiler.

The purpose of this feature is to allow us to enable the compiler to behave in it's old way if a developer did not specify a newer version.  It is not to enable the compiler to inspect from the code that you tried to use a newer feature and to warn you to specify a higher version.

I agree that such a feature might well be appropriate in the ide or perhaps even at build time:
"I have noticed that you have tried to use a struct tuple, use --langver:4.2 to enable this feature.

However, it's not really a scenario enabled by this switch, if you know what I mean.

I hope this makes sense